.github/workflows/lighthouse.yml

```diff
@@ -0,0 +1,20 @@
+name: Lighthouse check
+on: [ push ]
+jobs:
+  lhci:
+    name: Lighthouse
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      - name: Use Node.js 14.x
+        uses: actions/setup-node@v1
+        with:
+          node-version: 14.x
+      - name: Install & Build
+        run: |
+          yarn install
+          yarn build
+      - name: Lighthouse Check
+        run: |
+          npm install -g @lhci/cli@0.8.x
```

Comment:
We can probably add this lib to the dev dependencies and use `npx` to execute it locally from the local node_modules

Reply:
There is a problem with npx for now that always returns an error so I'll stick to npm install -g for now.